src/model/draft/common/mef-types.yang

```diff
@@ -43,11 +43,18 @@ module mef-types {
 
   revision YYYY-MM-DD {
```

Comment:
Date is missing

Reply:
I mentioned in my comments that I need to "build" the model once the changes are approved. That "build" will add the date.

Reply:
Committed version needs to have proper revision statements, including the date, so that people can use these models correctly.

Reply:
So long as the versions in src/model/ have the right revision etc, I am fine.  These are the versions people are looking at.